metashare/api/jobs.py

```diff
@@ -470,3 +473,44 @@ def populate_github_users(repository):
 
 
 populate_github_users_job = job(populate_github_users)
+
+
+def submit_review(*, user, task, data):
+    try:
+        task.refresh_from_db()
+        if task.commits and task.pr_is_open:
+            repo_id = task.project.repository.get_repo_id(user)
+            repository = get_repo_info(user, repo_id=repo_id)
+            pr = repository.pull_request(task.pr_number)
+            # The values in this dict are the valid values for the
+            # `state` arg to repository.create_status. We are not
+            # currently using all of them, because some of them make no
+            # sense for our system to add to GitHub.
+            state_for_status = {
+                # "": "pending",
+                # "": "error",
+                TASK_REVIEW_STATUS.Approved: "success",
+                TASK_REVIEW_STATUS["Changes requested"]: "failure",
+            }.get(data["status"])
```

Comment:
@wlonk My understanding of the desired functionality here is that every MS commit should get a `pending` status (probably with e.g. `description: "Waiting for MetaShare review"`) unless/until review is submitted. I think that would be straightforward to do -- is it a problem for most commits on a PR to remain "pending" indefinitely?

Reply:
I believe that @davisagli had said no, we should not use pending, when we talked on Thursday.

Reply:
Ah, ok -- I was going based on the Trello card, and don't recall that. 👍